app/commands/v2/discard_draft.rb

```diff
@@ -52,7 +52,7 @@ def send_live_to_draft_content_store(live)
 
         PresentedContentStoreWorker.perform_async(
           content_store: Adapters::DraftContentStore,
-          payload: { content_item: live.id, payload_version: event.id },
+          payload: { content_id: live.id, payload_version: event.id },
```

Comment:
I got a little confused here; I don't think this helps reduce the ambiguity. To me "content_id" implies the content_id UUID, which is the opposite of what you intended. Perhaps "content_item_id"?